public/homepage/stylesheets/glitch-notice.less

```diff
@@ -61,3 +61,98 @@
     }
   }
 }
+
+.glitch-cta.underlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.7);
+  z-index: 5;
+
+  &.hidden {
+    display: none;
+  }
+
+  .glitch-cta.content {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background: white;
+    padding: 80px;
+    text-align: center;
+    box-shadow: 10px 10px 0 0 #c1bfbf;
+
+    .cta-close {
+      position: absolute;
+      top: 32px;
+      right: 32px;
+      width: 14px;
+      height: 14px;
+      border: none;
+      background: inherit;
+      cursor: pointer;
+    }
+
+    .glitch-logo {
+      display: block;
+      width: 70px;
+      height: 54px;
+      margin: 0 auto 15px;
+      background-image: ~"url(/img/glitch.svg)";
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    h1 {
+      font-family: Arvo;
+      font-size: 30px;
+      font-weight: 900;
+      line-height: 1.2;
+      margin: 0;
+    }
+
+    p {
+      font-size: 14px;
+      font-family: 'Open Sans';
+      font-weight: 600;
+      line-height: 1.4;
+      margin: 30px auto;
+      padding: 0;
+    }
+
+    p:last-child {
+      margin-bottom: 0;
+    }
+
+    .export-button {
```

Comment:
can you also have a hover state to have a background of `#70e8b8`?

Reply:
I can, but that's the kind of stuff that should be in a comp, too because if that's what you want the button to do, I assume you want every button to do that, not just this one?